### PR TITLE
dependabot: add "github-actions"

### PR DESCRIPTION
**Pull request by HuijingHei**
Inspired by https://github.com/coreos/bootupd/pull/758#issuecomment-2441048372

### CODE REVIEW

**Review by travier — APPROVED** [review]

**Comment by travier** [comment]
We should probably add it to https://github.com/coreos/repo-templates/blob/main/dependabot/dependabot.yml as well so that it's propagated to all our repos

**Comment by HuijingHei** [comment]
> We should probably add it to https://github.com/coreos/repo-templates/blob/main/dependabot/dependabot.yml as well so that it's propagated to all our repos

Not sure if my understanding is correct, add it to dependabot.yml, and curious how this works, do you have related docs?

**Comment by travier** [comment]
I'm afraid that the only doc about how this repo works is in the README: https://github.com/coreos/repo-templates#coreos-repo-templates

The short version is that this generates the files from the templates and creates PRs to the repos to sync them.

File: dependabot/dependabot.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ updates:
 {{ dependabot_ecosystem }}
 {%- elif crate -%}
 cargo
+{%- elif github-actions -%}
+github-actions
```
[the remark this answers is not in the document]
Comment (travier) [reply dependabot/dependabot.yml:11]: i.e. we would directly include:

```
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
    open-pull-requests-limit: 3
```

right at the top of this list here

**Pull request merged**